sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -1774,7 +1777,7 @@ class DDLParserSuite extends AnalysisTest {
         allColumns = true))
 
     intercept("ANALYZE TABLE a.b.c COMPUTE STATISTICS FOR ALL COLUMNS key, value",
-      "mismatched input 'key' expecting {<EOF>, ';'}")
+      Some("PARSE_INPUT_MISMATCHED"), "syntax error at or near 'key'") // expecting {<EOF>, ';'}
```

Comment:
`expecting {<EOF>, ';'}` is not in the error message any more.

Reply:
I make it a comment in case in the future we are able to provide suggestion, and that may serves a reminder of what are really expected in the pre-changed code.
But I can remove it anyways.